### PR TITLE
feat(cli): add operator deregistration commands

### PR DESCRIPTION
**Pull request by mempirate**

### CODE REVIEW

**Review by thedevbirb — CHANGES_REQUESTED** [review]
Nice, can you modify the test for Symbiotic as well? The ignored test explains the instructions you need to do to run it.

**Review by estensen — APPROVED** [review]

File: bolt-cli/src/cli.rs [hunk 1]
```diff
@@ -228,6 +228,16 @@ pub enum EigenLayerSubcommand {
         expiry: U256,
     },
 
+    /// Deregister an EigenLayer operator from the bolt AVS.
```
Comment (merklefruit) [inline bolt-cli/src/cli.rs:231]: other commands are marked with "Step N", can you either add it here or remove it everywhere else to keep consistency in the doc comments?

File: bolt-cli/src/commands/operators.rs [hunk 2]
```diff
@@ -164,7 +164,46 @@ impl OperatorsCommand {
...
+
+                    info!(operator = %signer.address(), ?chain, "Deregistering EigenLayer operator");
+
+                    request_confirmation();
+
+                    let deployments = deployments_for_chain(chain);
+
+                    let bolt_avs_address = deployments.bolt.eigenlayer_middleware;
+                    let bolt_eigenlayer_middleware =
+                        BoltEigenLayerMiddleware::new(bolt_avs_address, provider.clone());
```
Comment (merklefruit) [inline bolt-cli/src/commands/operators.rs]: nit: is this clone needed?

File: bolt-cli/src/commands/operators.rs [hunk 3]
```diff
@@ -245,6 +284,46 @@ impl OperatorsCommand {
...
+
+                    let deployments = deployments_for_chain(chain);
+
+                    info!(operator = %signer.address(), ?chain, "Deregistering Symbiotic operator");
+
+                    request_confirmation();
+
+                    let middleware = BoltSymbioticMiddleware::new(
+                        deployments.bolt.symbiotic_middleware,
+                        provider.clone(),
```
Comment (merklefruit) [inline bolt-cli/src/commands/operators.rs]: same

**Review by merklefruit — APPROVED** [review]
good to go if test_eigenlayer_flow is working

**Comment by merklefruit** [comment]
Also can you update docs in the cli README to include usage of this command? 
Also ok to track it in a separate issue @mempirate

**Review by merklefruit — APPROVED** [review]

**Comment by mempirate** [comment]
@merklefruit added limited docs to README

**Review by thedevbirb — APPROVED** [review]

**Pull request merged**